docs/source-pytorch/accelerators/tpu_basic.rst

```diff
@@ -50,8 +50,8 @@ To run on different cores, modify the ``devices`` argument.
     trainer = Trainer(accelerator="tpu", devices=1)
     # run on multiple TPU cores
     trainer = Trainer(accelerator="tpu", devices=8)
-    # run on the 5th core
-    trainer = Trainer(accelerator="tpu", devices=[5])
+    # run on one specific TPU core: the 2nd core (index 1)
+    trainer = Trainer(accelerator="tpu", devices=[1])
```

Comment:
I changed the semantics here for consistency with other accelerators.

Before this was 1-based, now it's 0-based.